pages/release.md

```diff
@@ -1,15 +1,48 @@
 ---
 layout: page
-title: Releases
+title: Current Releases
 permalink: "/release/"
 ---
 
-Apache Fluo has several projects that are released separately on their own schedule.
+<div class="row">
+  <a href="/release/fluo-{{ site.latest_fluo_release }}">
+    <div class="col-sm-4 btn btn-info">
+      <h3>Fluo {{ site.latest_fluo_release }}</h3>
+      <p>Core project containing 'fluo' command to initialize<br> applications, start oracle and worker processes, and<br>manage running applicaitons.</p>
+    </div>
+  </a>
+  <a href="/release/fluo-yarn-{{ site.latest_fluo_yarn_release }}">
+    <div class="col-sm-4 btn btn-warning">
+      <h3>Fluo YARN {{ site.latest_fluo_yarn_release }}</h3>
+      <p>Project containing 'fluo-yarn' command that launches<br>Fluo applications in YARN after they have been<br>initialized using 'fluo' command</p>
+    </div>
+  </a>
+  <a href="/release/fluo-recipes-{{ site.latest_recipes_release }}">
+    <div class="col-sm-4 btn btn-success">
+      <h3>Fluo Recipes {{ site.latest_recipes_release }}</h3>
+      <p>Libraries containing common code that build on the<br>Fluo API to offer complex transactional updates or<br>provide additional utilities<br></p>
+    </div>
+  </a>
+</div>
 
-{% for release in site.categories.release %}
-{% unless release.historical or release.draft %}
-* [{{ release.version }}]({{ site.baseurl }}/release/{{ release.version }}/) - {{ release.date | date_to_string }}
-{% endunless %}
+## Release Archive
+
+{% assign visible_releases = site.categories.release | where:"draft",false | where:"historical",false %}
```

Comment:
How long did it take you to figure out that liquid filter syntax? :smiley_cat:

Reply:
It took awhile. :-)